render-release-notes.sh

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
+
+me=$(basename $0)
+
+function showUsage() {
+	cat<<-EOF
+Usage: $me --installer <msi|inno|nuget>
+   or: $me [--output <dir> [--css <path>]]
```

Comment:
Do we really only accept `--css` when `--output` was specified?

Also, I find it much easier to go with the single-parameter form: `[--output=<dir>] [--css=<path>]`. This script is meant primarily for consumption by scripts, after all.